### PR TITLE
Telegram and Github accounts can differ; Added 'Unassigned' section

### PR DESCRIPTION
**Pull request by nvc8996**
Yaml format is now used for configuration.
Telegram and Github accounts can be different.
(#4 )
Added 'Unassigned' section. (#22 )

### CODE REVIEW

File: config.yml.example [hunk 1]
```diff
@@ -1,10 +1,10 @@
 # Github Personal Access Token (https://github.com/settings/tokens) with rights:
```
Comment (talyguryn) [inline config.yml.example]: it is better to use `example` suffix not at the end of filename to allow IDE detect and highlight code syntax correctly.

`config.example.yml` would be great.

**Review by talyguryn — CHANGES_REQUESTED** [review]

File: src/index.js [hunk 2]
```diff
@@ -25,23 +29,44 @@ const TRIM_PR_NAME_LENGHT = 35;
...
 const CARDS_QUERY = require('./queries/cards');
 const ISSUE_QUERY = require('./queries/issue');
 const PR_QUERY = require('./queries/pr');
+const { Console } = require('console');
+
+/**
+ * Parse mention list to string
+ * and build mention map that links github to telegram usernames.
+ */
+function parse_mention(mentions) {
```
Comment (neSpecc) [inline src/index.js]: Can we move valued/utility code fragments to separate files? The `index.js` should be just an entry point.

File: src/index.js [hunk 3]
```diff
@@ -25,23 +29,44 @@ const TRIM_PR_NAME_LENGHT = 35;
...
+  var mention = [];
```
Comment (neSpecc) [inline src/index.js]: please, use our code style and ESLint linter for that.

**Review by neSpecc — CHANGES_REQUESTED** [review]

File: config.yml.example (hunk 1 above)
Comment (nvc8996) [reply config.yml.example]: Thanks for suggestion.

File: src/index.js (hunk 2 above)
Comment (nvc8996) [reply src/index.js]: Thank you. I'm gonna think about it.

File: src/index.js (hunk 3 above)
Comment (nvc8996) [reply src/index.js]: Thanks for advice. I've updated.

File: src/index.js (hunk 2 above)
Comment (neSpecc) [reply src/index.js]: you can start with moving this method to the utility file

**Review by robonetphy — APPROVED** [review]

**Review by robonetphy — APPROVED** [review]

**Review by neSpecc — APPROVED** [review]

**Pull request merged**